Modelica/Resources/CMakeLists.txt

```diff
@@ -17,11 +17,13 @@ set(MODELICA_UTILITIES_INCLUDE_DIR "${MODELICA_RESOURCES_DIR}/C-Sources")
 include(BuildProjects/CMake/Modelica_platform.cmake)
 include(BuildProjects/CMake/Modelica_utilities.cmake)
 
-set(
-  CMAKE_INSTALL_PREFIX "${MODELICA_RESOURCES_DIR}"
-  CACHE PATH
-  "Library installation prefix path (don't change)" FORCE
-)
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
```

Comment:
I think there should be comment referencing the fact that this variable is automatically set, so that people reading the script don't have to search for the documentation.